src/style/blocks/stars.css

```diff
@@ -0,0 +1,19 @@
+.Stars {
+  display: flex;
+}
+
+.Stars__Star {
+  height: 16px;
+  width: 16px;
+  background-image: url(/images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.Stars__Star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.Stars--4 .Stars__Star:not(:last-child) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
Please, pay attention to styles of stars, reuse it from previoy task. It should work for all modificators (like, I will change in HTML Stars--3 - and should be just 3 golden stars